WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -372,6 +372,13 @@ def time_until_competition
     }
   end
 
+  def dues_formatting
+    converted = Money.us_dollar(params[:value]).exchange_to(params[:to_currency])
+    render json: {
+      html: converted.format,
```

Comment:
Is the result of the `.format` call actual HTML?

Reply:
Otherwise, the name of the key is extremely misleading.

Reply:
> 

nope, it isn't HTML :p